circle.yml

```diff
@@ -0,0 +1,55 @@
+machine:
+  environment:
+    ARCH: "i686"
+    BUILDOPTS: "-j3 VERBOSE=1 FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1"
+    TESTSTORUN: "all"
+    JULIA_CPU_CORES: 2
```

Comment:
Actually, we might be able to set up the parallelism by using different `TESTSTORUN` for the different containers. So for example, one could be running the linear algebra tests while another does libgit2, etc. I'm really familiar with how the `choosetests` thing works but that seems like it could be doable, right?